Runtime/Internal/PeerConnection.cs

```diff
@@ -361,15 +362,18 @@ private void EstablishEvents()
     /// </summary>
     /// <param name="signaling"></param>
     /// <param name="configuration"></param>
-    public void SetUp(ISignaling signaling, RTCConfiguration configuration)
+    public void SetUp(ISignaling signaling, RTCConfiguration configuration, bool addTransceiver = false)
```

Comment:
why do we need to optionally add transceivers?

Reply:
if(addTransceiver)
      {
        _pc.AddTransceiver(TrackKind.Video);
        _pc.AddTransceiver(TrackKind.Audio);
      }
     As i added this check to avoid adding transceiver for publisher to fix issues with simulcast publisher. For subscriber, it should be as is.